### PR TITLE
hardening/635 update the needed dependencies

### PR DESCRIPTION
**Pull request by pcoello25**
* Fixes issue https://github.com/telefonicaid/fiware-cygnus/issues/635
* No test are required
* Assignee @frbattid

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -11,3 +11,4 @@
 - [FEATURE] Enable per-user or default Hive database creation in OrionHDFSSink(#516)
 - [HARDENING] Document the service_as_namespace OrionHDFSSink parameter (#692)
 - [HARDENING] Add a piece of documentation about multitenancy (#607)
+- [HARDENING] Add a piece of documentation about dependencies (#635)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: I would say "Add a piece of documentation about required dependencies to be manually installed when building and installing without dependencies"

File: doc/installation_and_administration_guide/install_from_sources.md [hunk 2]
```diff
@@ -106,25 +112,41 @@ If the dependencies are included in the built Cygnus package, then nothing has t
 [Top](#top)
 
 ##<a name="installdeps"></a>Installing dependencies
+###<a name="cygnusdeps"></a>Cygnus dependencies
+These are the packages you will need to install under `APACHE_FLUME_HOME/plugins.d/cygnus/libext/` **if you did not included them in the Cygnus package**:
+
+|       Cygnus dependencies        |
+|:---------------------------------------:|
+|       flume-ng-core-1.4.0.jar      |
```
Comment (frbattid) [inline doc/installation_and_administration_guide/install_from_sources.md]: A second coulmn regarding the version is required. For instance:

| Cygnus dependencies | Version |
|---|
| flume.ng-core | 1.4.0 |
| etc | |

Please, apply this comment to rest of the document.

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (pcoello25) [inline CHANGES_NEXT_RELEASE]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/35d7aac2157f5ecb31083f68d30a4e970cb95c15

File: doc/installation_and_administration_guide/install_from_sources.md (hunk 2 above)
Comment (pcoello25) [inline doc/installation_and_administration_guide/install_from_sources.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/35d7aac2157f5ecb31083f68d30a4e970cb95c15

**Comment by frbattid** [comment]
LGTM

**Pull request merged**